algorithms/src/sorting/impl/Kokkos_SortImpl.hpp

```diff
@@ -412,7 +412,7 @@ sort_device_view_with_comparator(
 // Note with HIP unified memory this code path is still the right thing to do
 // if we end up here when RocThrust is not enabled.
 // The create_mirror_view_and_copy will do the right thing (no copy).
-#ifndef KOKKOS_ENABLE_IMPL_HIP_UNIFIED_MEMORY
+#ifndef KOKKOS_IMPL_HIP_UNIFIED_MEMORY
```

Comment:
Can you move the `memspace` type alias above to be in this `ifndef`?
I'm getting warnings on this:

```
/kokkos/algorithms/unit_tests/../src/sorting/./impl/Kokkos_SortImpl.hpp:411:9: warning: unused type alias 'MemSpace' [-Wunused-local-typedef]
  411 |   using MemSpace = typename ViewType::memory_space;
      |         ^
 ```

Reply:
Yes, I'll do it.